hrjobcontract/views/contractList.html

```diff
@@ -3,12 +3,12 @@ <h3 class="header-bar header-bar-primary" ng-show="contractCurrent.length">Curre
     <li class="{{prefix}}list-contract-item" hrjc-loader hrjc-loader-show="true" ng-class="{'ng-active': contractLoaded}" ng-repeat="contract in contractCurrent track by contract.id" ng-controller="ContractCtrl" ng-include src="'contract.html'"></li>
 </ul>
 
-<h3 class="header-bar header-bar-danger-light" ng-show="contractPast.length">Past Contracts:</h3>
+<h3 class="header-bar header-bar-danger" ng-show="contractPast.length">Past Contracts:</h3>
 <ul class="{{prefix}}list-contract">
     <li class="{{prefix}}list-contract-item" hrjc-loader hrjc-loader-show="true" ng-class="{'ng-active': contractLoaded}" ng-repeat="contract in contractPast track by contract.id" ng-controller="ContractCtrl" ng-include src="'contract.html'"></li>
```

Comment:
please fix the indentation in the whole file

Reply:
ok. I will change this